scss/_patterns_navigation.scss

```diff
@@ -297,17 +309,31 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
 
   // navigation items
   .p-navigation__items {
-    @extend %vf-navigation-separator;
-
     display: none; // hidden by default on mobile (expands is a dropdown)
     list-style: none;
     margin: -1px 0 0 0;
     padding: 0;
     position: relative;
 
-    // remove navigation separator for the first navigation items list
-    &:first-child::before {
-      content: none;
+    // remove navigation separator for the last navigation items in last navigation items list
+    &:last-child {
```

Comment:
Hmm okay, I'll try to see if I can make this work

Reply:
Upon trying to implement this I realised that the situation for the override is very specific: 
- the last `.p-navigation__items`
  - the last `.p-navigation__item / .p-navigation__item--dropdown-toggle`
    - the last `.p-navigation__link`
  - the last `.p-navigation__item--dropdown-toggle`
    - the last `li`
      - `.p-navigation__dropdown-item`

And that implementing this would require restructuring what I think would be a great chunk of code in the file.